config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
```

Comment:
Kubelet port currently listens on port 10250, but is secured by mutual TLS to the Kubernetes master. There is currently no way to auth to this without managing certificates to talk to kubelet. So in Kubernetes & OpenShift the current way to manage this is to open a read-only port on the kubelet - this is done by default installations on port 10255, hence this port number here. Once auth to the kubelet is sorted properly in kubernetes/openshift, this port will change to 10250 & use whatever mechanism they put in place for auth.